mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -428,6 +430,21 @@ class BackgroundTasksManager : BroadcastReceiver() {
             )
         }
 
+        fun enqueueNotificationAction(context: Context, action: CloudNotificationAction) {
+            val split = action.action.split(":", limit = 3)
```

Comment:
Ideally that kind of parsing should happen in `CloudNotificationAction` itself...maybe with a sealed class, I'll have a look at it.

Reply:
For your consideration (also dealing with the comment above): https://github.com/maniac103/openhab.android/commits/notification-enhancements/